Privacy Policy.md

```diff
@@ -0,0 +1,60 @@
+
+When you visit our website, we may collect information about your use of the website, including your IP address, which is stored for no longer than 45 days. We may use this information to operate, maintain, and improve the website.
+
+If you submit information through the Contributor and Workstream Participant Agreement, including your name, city, email, and Github username, WHATWG will use this data to comply with the Intellectual Property Rights policy, including licensing obligations for contributions and participation in workstreams. WHATWG will retain this data for as long as necessary to track all IPR obligations covering WHATWG standards.
+
+We retained email addresses for as long as users were subscribed to the email distribution list or for a brief period after we shut down the email distribution list.
+
+If you comment on a blog post on blog.whatwg.org, your comment will be published with the information you submit, including your name and email address. WHATWG will only use the information you submit to publish your comment. To delete the name and e-mail address associated with a published comment, you can contact sg@whatwg.org.
+
+Notwithstanding the data retention policies described above, we may retain your personal information where necessary to comply with applicable legal obligations.
+
+This website is not intended for use by children. If we learn that we have collected the personal information of a child under 13, or equivalent minimum age depending on jurisdiction, we will take steps to delete the information as soon as possible.
+
+## Disclosure to Third Parties
+
+We may disclose your personal information to our service providers to perform certain services on our behalf such as email service delivery. We may also disclose your personal information if obligated by law, legal process, litigation and/or requests from public and governmental authorities within or outside your country of residence. Finally, we may disclose your personal information if we determine that for purposes of national security, law enforcement, or other issues of public importance, disclosure is necessary or appropriate.
+
+This site may contain links to third-party websites, products, and services. Information collected by third parties, which may include your personal information, is governed by their privacy policies and practices.
+
+## International Data Transfers
+
+If you are in the European Economic Area (EEA), you should be aware that your personal information may be transferred outside the EEA.
+
+## Protection of Personal Information
+
+We have appropriate technical and organizational security measures in place to prevent accidental or unlawful destruction or accidental loss, alteration, unauthorized disclosure or access.
+
+## Cookies and Other Technologies
+
+Our website uses cookies to retain your logged-in state.  It does not use other tracking technologies and it does not use automated decision-making.
```

Comment:
"Our website" here refers to blog.whatwg.org and wiki.whatwg.org only, I believe. As a reader, I'd feel more comfortable if that were spelled out explicitly.

Reply:
Yeah, and it's also only applicable if you have an account.

Reply:
Do you have a wording suggestion for this? I was pretty happy with this simple statement and that it's clear it only applies to logging in, and spelling out exactly which domains it can happen for means we have to update the privacy policy if we make the blog or wiki static, or if (less likely) we have some other system where it's possible to log in.

Reply:
For the first sentence:

> blog.whatwg.org and wiki.whatwg.org support accounts and use cookies to retain your logged-in state, provided you have an account and are logged in.

Reply:
I'd prefer @annevk's suggestion. If we intentionally want to leave it vague, then maybe something like

> Some parts of our website allow logging in. In such cases, our website uses cookies to retain...

Reply:
I've done with @domenic's suggestion in https://github.com/whatwg/sg/pull/120/files#r419731787